xml/System.Xaml/XamlLanguage.xml

```diff
@@ -100,7 +100,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- A factory method is a [!INCLUDE[xaml2009](~/includes/xaml2009-md.md)] feature for identifying an alternative technique that constructs the underlying type for a XAML type. The `Arguments` specify the input of a factory method and enable specifying that information in markup. The default techniques, in absence of a factory method, are to attempt to find a parameterless constructor and then attempt to find an identified type converter for type, member, or destination type.  
+ A factory method is a XAML2009 feature for identifying an alternative technique that constructs the underlying type for a XAML type. The `Arguments` specify the input of a factory method and enable specifying that information in markup. The default techniques, in absence of a factory method, are to attempt to find a parameterless constructor and then attempt to find an identified type converter for type, member, or destination type.  
```

Comment:
```suggestion
 A factory method is a XAML 2009 feature for identifying an alternative technique that constructs the underlying type for a XAML type. The `Arguments` specify the input of a factory method and enable specifying that information in markup. The default techniques, in absence of a factory method, are to attempt to find a parameterless constructor and then attempt to find an identified type converter for type, member, or destination type.  
```

Reply:
Oh, literally the word "XAML 2009". I searched for one of the md files with 2009 in their name and only found the MSBuild one.